src/GHC.hs

```diff
@@ -29,7 +29,7 @@ defaultKnownPackages =
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghci
     , ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc
     , hpcBin, integerGmp, integerSimple, iservBin, libffi, mkUserGuidePart
-    , parallel, pretty, primitive, process, rts, runGhc, stm, templateHaskell
+    , pretty, process, rts, runGhc, templateHaskell
```

Comment:
There seems to be a misunderstanding. I am not asking how to exclude _parallel_, _primitive_ and _stm_ from the tarball. This is already done (because that was also the case for the _make_ generated tarball). 

The reason for the removal of them from _defaultKnownPackages_ is that  `./build.sh --flavour=quick` on the extracted sources is failing, because _parallel_, _primitive_ and _stm_ are intentional not part of the tarball.